forward_script/app.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+use Silex\Application;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Process\Process;
+use Wikimedia\ForwardScript\PageInformationCollector;
+use Wikimedia\ForwardScript\ApplicationException;
+
+require __DIR__.'/vendor/autoload.php';
+
+
+$app = new Application();
+
+// Settings
+$app["cache_dir"] = __DIR__."/cache";
```

Comment:
Single quotes are preferred over doubles quotes in MediaWiki's coding conventions